### DISCUSSION TITLE
The mod operation is not supported when XLS_IR is converted to Z3. Is there a solution

### DISCUSSION THREAD

**Discussion by zhang0504** [google/xls#1670] [Q&A]
UNIMPLEMENTED: Unhandled node for conversion from XLS IR to Z3:

**Comment by hzeller** [comment]
You asked that in https://github.com/google/xls/issues/1669, but that is missing some crucial information. Here, you also did not provide any information.

Create a simple DSLX input file, the commands you used to operate on these such as `ir_converter_main` or `opt_main`  and the error message it prints. This allows anyone looking at the issue to reproduce it and can potentially provide help.
